app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt

```diff
@@ -47,24 +41,14 @@ fun ConnectScreen(
 @Stable
 @Composable
 fun LogoHeader() {
-    CenterRow {
+    CenterRow(
+        modifier = Modifier.padding(vertical = 25.dp),
```

Comment:
Why specifically the 25 dp?

Reply:
Graphic design is just inputting random numbers until it feels right

Reply:
The odd number irks me, but oh well.